org/teams.yaml

```diff
@@ -355,19 +325,17 @@ teams:
   Working Group Leads:
     description: Working group leads - for easy communication in github
     members:
-      - ericvn
-      - esnible
-      - frankbu
-      - howardjohn
-      - jacob-delgado
-      - liminw
-      - linsun
-      - lizan
-      - mandarjog
-      - Monkeyanator
-      - myidpt
-      - nrjpoddar
-      - stevenctl
+      - costinm # Env
+      - ericvn # T&R
+      - GregHanson # PSWG
+      - howardjohn # Control Plane
+      - justinpettit # PSWG
+      - lei-tang # Security, E&T
+      - lizan # Data Plane, Security
```

Comment:
Is lizan still active? should we nominate a new person? @kyessenov should be the guy if he want.

Reply:
cc @lizan

Reply:
@zirain can you please ping @lizan internally and ask him to make a decision here?

Reply:
IIRC, @lizan leave Tetrate for a while?

Reply:
/cc @therealmitchconnors as Lizan is now at Aviatrix.

Reply:
/cc @therealmitchconnors as Lizan is now at Aviatrix.